dbms/src/Storages/DeltaMerge/Index/LocalIndexInfo.cpp

```diff
@@ -159,31 +159,6 @@ LocalIndexInfosChangeset generateLocalIndexInfos(
     std::vector<ComplexIndexID> newly_added;
     std::vector<ComplexIndexID> newly_dropped;
```

Comment:
Since the vector index is not defined in the ColumnInfo, we can use `IndexID` instead of `ComplexIndexID`

Reply:
done